io.github.technic.e2designer.yaml

```diff
@@ -0,0 +1,24 @@
+app-id: "io.github.technic.e2designer"
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: e2designer
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
```

Comment:
If your app only opens individual files, it doesn't need filesystem permissions at all - they'll be passed through with Open/Save dialog.

Reply:
It opens the whole directories.